.eslintrc.json

```diff
@@ -22,6 +23,8 @@
                 "@typescript-eslint"
             ],
             "rules": {
+                "@typescript-eslint/triple-slash-reference": "off",
+                "spaced-comment": ["error", "always", { "markers": ["/"] }],
```

Comment:
Why are these changes necessary?

Reply:
Without the /// Reference will be marked as linting error and will be auto fixed to // / and then the PipelineStages will not be imported and then the definitions will be broken.  Triple Slash is a typescript specific operation to import the definitions from another file.
With this change the Definitions will work.